### ISSUE TITLE
Mark all emails read

### ISSUE THREAD

**Issue by kicohen** [ABTech/abtt#343]
Can there be a Mark All Read button on the emails page?

**Comment by timparenti** [comment]
Considering that the unread state of a given message is global for _all_ users, this strikes me very much as a Bad Idea.  Currently, at least _someone_ has to somewhat actively process each message — even if it's just skimming the metadata — which ensures nothing important is missed.

That said, checkboxes to mark/unmark a specific set of selected messages _en masse_ could be quite useful, though.

**Comment by kicohen** [comment]
I would agree if this was our primary inbox, however, all of exec uses the gmail inbox as our primary source of mail. Honestly we could probably just get rid of read/unread on emails but let's leave it for now.

**Comment by EmNewman** [comment]
+1 for checkboxes, with a "select all" button :)

**Comment by EmNewman** [comment]
but also yeah, gmail inbox is definitely where all the reading/data-digesting happens

**Comment by kicohen** [comment]
I'm not going to mark all 750 emails on tracker as read individually, there should be a mark all emails as read button.
